### ISSUE TITLE
The "(required)" contact form text in the Nucleare theme is "(obligatoire)" and I tried changing it, but it breaks the entire contact form? 

### ISSUE THREAD

**Issue by Magew** [getgrav/grav-plugin-form#31]
I changed "obligatoire" to "required" in templates/forms/field.html.twig and it broke the entire contact form. I reverted the word change and it fixed it. What am I doing wrong?

{{ field.validate.required in ['on', 'true', 1] ? '(obligatoire)' }}

**Comment by flaviocopes** [comment]
I just tried installing that theme and editing that bit, it works fine:

![screen shot 2015-12-18 at 10 18 24](https://cloud.githubusercontent.com/assets/11783388/11893411/ba1c0d74-a570-11e5-969d-4edbaa91f69a.png)

If you tell the error you get, maybe I can give an hint.

**Comment by Magew** [comment]
I receive no error, the input forms just disappear and so does the (required) and all that shows up is "Name" "Email" "Message" "Captcha"

**Comment by flaviocopes** [comment]
Something's wrong with your edits it seems. Paste the entire page code that gives you the error.

**Comment by Magew** [comment]
Thanks for the quick response.

I just figured it out and it ended up being what I thought it was. I had been editing the files using TextEdit on OSX because FileZilla wasn't complying with my system settings and opening them in my default editor for .twig files.

Every time I saved with TextEdit it likely was adding some sort of rich text format or something of that sort to the file and breaking it completely.

All is working now, dumb mistake by me.
